crates/tauri-bundler/src/bundle/windows/nsis/mod.rs

```diff
@@ -51,6 +51,16 @@ const NSIS_REQUIRED_FILES: &[&str] = &[
   "Include/nsDialogs.nsh",
   "Include/WinMessages.nsh",
 ];
+#[cfg(target_os = "windows")]
+const NSIS_PLUGIN_FILES: &[&str] = &[
+  "NSISdl.dll",
+  "StartMenu.dll",
+  "System.dll",
+  "nsDialogs.dll",
+  "nsis_tauri_utils.dll",
+];
+#[cfg(not(target_os = "windows"))]
+const NSIS_PLUGIN_FILES: &[&str] = &[];
```

Comment:
hmmm, this doesn't sound right, this makes it so that signing works on Windows but not on other platforms, we should at least sign `nsis_tauri_utils.dll` on other platforms

Reply:
But other platforms doesn't have DLLs. I'm not sure what do you suggest we sign on other platform?

Reply:
other platforms can cross-build a windows NSIS installer, and these dlls exist, see https://tauri.app/distribute/windows-installer/#install-nsis

Reply:
Fixed